packages/flutter/lib/src/material/selectable_text.dart

```diff
@@ -620,6 +628,20 @@ class _SelectableTextState extends State<SelectableText> implements TextSelectio
     });
   }
 
+  void _handleFocusChanged() {
+    if (!_effectiveFocusNode.hasFocus
+        && SchedulerBinding.instance.lifecycleState == AppLifecycleState.resumed) {
+      // We should only clear the selection when this SelectableText loses
+      // focus while the application is currently running. It is possible
+      // that the application is not currently running, for example on desktop
+      // platforms, clicking on a different window switches the focus to
+      // the new window causing the Flutter application to go inactive. In this
+      // case we want to retain the selection so it remains when we return to
+      // the Flutter application.
```

Comment:
So @gspencergoog when the application is backgrounded, any focused FocusNode loses focus? Just making sure that's how it works and is intended.

Reply:
For more context on this, as of https://github.com/flutter/flutter/pull/142930, the `FocusManager` responds to app lifecycle changes.

Reply:
This looks correct to me then, thanks for the context 👍